cmd/kubelet/app/server.go

```diff
@@ -281,6 +281,7 @@ func (s *KubeletServer) Run(_ []string) error {
 		TLSOptions:                     tlsOptions,
 		ImageGCPolicy:                  imageGCPolicy,
 		Cloud:                          cloud,
+		NodeStatusUpdateFrequency: s.NodeStatusUpdateFrequency,
```

Comment:
I know that this formatting is pretty strange - but this is what gofmt expects.
I've reported an issue for it: https://github.com/golang/go/issues/10392